### PR TITLE
Add interrogate

### PR DESCRIPTION
**Pull request by and-semakin**
### Resource description
`interrogate` is the tool to measure docstring coverage (like for tests, but for documentation). It supports configuring via `pyproject.toml`.
### By submitting this pull request I confirm I've read and complied with the below requirements:
- [x] I have read and understood the [Contribution Guidelines](https://github.com/carlosperate/awesome-pyproject/blob/master/contributing.md)
- [x] I am making an individual pull request for each entry
- [x] I have added a useful title to the commit
- [x] I have added a useful title to this PR
- [x] I have added the new entry in alphabetical order
- [x] I have used the following format:
  ```
  - [Resource Title](link) - Resource short Description (2 lines or less in total)
  ```
- [x] The new entry meets one of these categories:
    - Is configured via its own tool sub-table in the pyproject.toml file (i.e., `[tool.xxx]`)
    - Is a project template using the pyproject.toml file
    - Is an article about the pyproject.toml file
    - Is a link to a project discussion about pyproject.toml support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ Python project templates or project generators supporting `pyproject.toml`.
 - [Autohooks](https://github.com/greenbone/autohooks/blob/master/README.md#1-choosing-an-autohooks-mode) - Library for managing git hooks.
 - [Commitizen](https://commitizen-tools.github.io/commitizen/config/) - Create commiting rules for projects, auto bump versions and auto changelog generation.
 - [django-pyproject](https://github.com/Ceterai/django-pyproject) - Django package to store some or all of your settings in your pyproject.toml file.
+- [interrogate](https://github.com/econchick/interrogate) - Interrogate a codebase for docstring coverage.
```
Comment (carlosperate) [inline README.md:91]: This is great, thanks for the submission @and-semakin!
Normally I like to link to the part of the documentation that lists how to configure it with the pyproject.toml format. Is there a link like that in the docs for interrogate?

Comment (and-semakin) [reply README.md:91]: Hi @carlosperate! Unfortunately, there is no separate section in docs about configuring it with `pyproject.toml`. The most precise link I was able to find is this:
* https://github.com/econchick/interrogate#other-usage
* https://interrogate.readthedocs.io/en/latest/#other-usage

Comment (carlosperate) [reply README.md:91]: Yeah, https://interrogate.readthedocs.io/en/latest/#other-usage will do 👍

Comment (and-semakin) [reply README.md]: Done :)

Comment (carlosperate) [reply README.md]: Great, we are ready to merge, thanks!

**Pull request merged**